tools/python/m2k-osc/ain_tab.py

```diff
@@ -0,0 +1,199 @@
+import matplotlib
+import numpy as np
+import tkinter as tk
+from tkinter import ttk
+import matplotlib.figure as  fg
+import libm2k
+from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg, NavigationToolbar2Tk
+from matplotlib import style
+import threading
+import time
+import ctx_tab
+import aout_tab
+LARGE_FONT = ("Verdana",12)
```

Comment:
if you remove these fonts and leave them in the app part to choose, would it work?